recipes/ezca/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "ezca" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/cds/python-ezca/-/archive/{{ version }}/python-ezca-{{ version }}.tar.gz
+  sha256: be44414000dfbd362b02afcc92b871760f22cfac0f94a7a2c56ede54f07c21bd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

This can be simplified now:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```